app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -23,14 +23,14 @@ sealed class Event {
         object SetAsDefault : GrowthData("xgpcgt")
 
         /**
-         * Event recording the first time Firefox has been resumed in a 24 hour period.
+         * Event recording that a URI has been loaded.
```

Comment:
```suggestion
         * Event recording that a URI has been loaded during the first day of install
```

Reply:
I had tried to write a more generalized version in anticipation of future requirements changes, since this is the second time logic has shifted around when we are capturing a URI load event. That said, because of the current structure of the code where we are `track`ing specific events, I don't think a generalized version will work without further design changes. 

Just adding context, because I think your suggestion here is correct.

Reply:
Understood, thank you for the added context 🙂 